### ISSUE TITLE
Empty relative_offsets?

### ISSUE THREAD

**Issue by GeoffreyFrogeye** [claroty/access_parser#12]
I'm trying to open [this file](https://servicetool.blob.core.windows.net/release/Parameters/\$_parameters_HeatPump.par). It can parse some tables fine, but some tables fail on the following:

```
IndexError: list index out of range
> /home/geoffrey/.local/lib/python3.10/site-packages/access_parser-0.0.4-py3.10.egg/access_parser/access_parser.py(344)_parse_dynamic_length_data()
    342                 if i in relative_record_metadata.variable_length_jump_table:
    343                     jump_table_addition += 0x100
--> 344             rel_start = relative_offsets[i]
    345             # If this is the last one use var_len_count as end offset
    346             if i + 1 == len(relative_offsets):
```

Seems like `relative_offsets` is empty at this point. I'm not sure what this could be? mdbtools is able to extract those without issue

On this particular table (`PAR-file version`), I also got `ERROR:Failed to parse bool field, Column not found in null_table column: Vrijgegeven , column id: 12 , null_table: [False, False, False, True, True, False, True, True]`, but it doesn't happen for all tables (`Parameterlijst_V37`).

In case this might be of help, here's [a smaller file](https://servicetool.blob.core.windows.net/release/Parameters/\$_parameters_APSloader.par) where the same problem happens.

**Comment by ur1katz** [comment]
Hi, Can you please check if this resolves your problem? #13

**Comment by GeoffreyFrogeye** [comment]
Hi! This does solve a lot of the problems I have with my files already, tahnks a lot!

However, it doesn't fix all of them. I still get the error with the table `L_M_Status` of [this file](https://servicetool.blob.core.windows.net/release/Parameters/$_parameters_HRU250-300.par) (it also happens with [the original file I gave](https://servicetool.blob.core.windows.net/release/Parameters/$_parameters_HeatPump.par) in the table `Parameterlijst_V31` but it's bigger, has warnings, and I don't need that table there).
In this case though, `relative_offsets` is not empty, its length is 6 whereas `relative_records_column_map` is length 7.

If I understand correctly (but I could be completely off the mark, so feel free to ignore this paragraph), the case where all the columns were of dynamic length was already covered, you covered the case if none is, but the case where only some are is not. How to know which are variables and which aren't is a question I couldn't find the answer ; it doesn't seem to be in `relative_records_column_map[x].column_flags` at least.
